### PR TITLE
Account settings  backend & frontend integration

### PR DESCRIPTION
**Pull request by Eneshjakhar**
Changes made:
- adjusted Home page css to have better navigation bar
- changes colligo logo
- added frontend for account setttings
- added AccountSettings.css
- added AccountSettings.html

### CODE REVIEW

File: DiscordCode/FrontEnd/AccountSettings.html
Comment (ez4papiz1) [inline DiscordCode/FrontEnd/AccountSettings.html:1]: Looks good

File: DiscordCode/FrontEnd/css/AccountSettings.css
Comment (ez4papiz1) [inline DiscordCode/FrontEnd/css/AccountSettings.css:1]: Looks good

File: DiscordCode/FrontEnd/css/home.css
Comment (ez4papiz1) [inline DiscordCode/FrontEnd/css/home.css:1]: Looks good

File: DiscordCode/FrontEnd/images/logo.png
Comment (ez4papiz1) [inline DiscordCode/FrontEnd/images/logo.png:1]: Looks good

**Review by ez4papiz1 — APPROVED** [review]
All the changes seem correct. Should be able to merge without conflict

File: DiscordCode/BackEnd/routes/AccountSettings.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const user = require('./models/User');
+
+mongoose.connect('mongodb://localhost:27017/colligo');
```
Comment (jordanp2002) [inline DiscordCode/BackEnd/routes/AccountSettings.js:5]: Should be changed to createConnection from ".connect"

File: DiscordCode/BackEnd/routes/AccountSettings.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+const db = mongoose.connection;
+
+const router = express.Router();
+
+router.post('/', async (req, res) => {
```
Comment (jordanp2002) [inline DiscordCode/BackEnd/routes/AccountSettings.js:10]: Should be sending json data back so frontend can display this messages to the user.

**Review by jordanp2002 — CHANGES_REQUESTED** [review]

File: DiscordCode/BackEnd/routes/AccountSettings.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const user = require('./models/User');
+
+mongoose.connect('mongodb://localhost:27017/colligo');
+const db = mongoose.connection;
+
+const router = express.Router();
+
```
Comment (jordanp2002) [inline DiscordCode/BackEnd/routes/AccountSettings.js:9]: Should be rendering html in the route. Eg) res.render('account_settings')

**Pull request closed without merging**